tracer/src/Datadog.Trace/Tags.cs

```diff
@@ -242,6 +242,11 @@ public static class Tags
         /// </summary>
         internal const string KafkaOffset = "kafka.offset";
 
+        /// <summary>
+        /// The consumer group that consumed the message
+        /// </summary>
+        internal const string KafkaConsumerGroup = "kafka.group";
```

Comment:
FYI, each time we will add a new tag in the near future, we need to make sure it is already handled by the TagRemapping done on the backend side. This one already existed in Java so we're good. [Here's the code for later reference](https://github.com/DataDog/logs-backend/blob/prod/domains/event-platform/apps/processing/src/main/java/com/dd/logs/processing/processors/track/SpansTagRemapper.java#L95)